### PR TITLE
test: fix parallel test run

### PR DESCRIPTION
**Pull request by pcheremu**
# What ❔

Fix conflict in tests for parallel run

## Checklist

- [x] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [x] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.

### CODE REVIEW

**Comment by hedgar2017** [comment]
Please wait for an @antonbaliasnikov's review.

File: cli-tests/tests/output-dir.test.ts [hunk 1]
```diff
@@ -60,45 +69,52 @@ describe("Set of --output-dir tests", () => {
...
   });
 
   //id1812 - different behaviour on CI on Linux
   if (os.platform() !== 'linux') {
     describe(`Run ${zksolcCommand} with --output-dir - output-dir - wrong permissions`, () => {
-      createDirectory(paths.pathToReadOnlyOutputDir)
-      changeDirectoryPermissions(paths.pathToReadOnlyOutputDir, 'r');
-      const args = [`${paths.pathToBasicSolContract}`, `--bin`, `--output-dir`, `${paths.pathToReadOnlyOutputDir}`];
+      const tmpDirZkSolc = createTmpDirectory();
+      // const tmpDirSolc = createTmpDirectory();
```
Comment (antonbaliasnikov) [inline cli-tests/tests/output-dir.test.ts:115]: Please, remove comments if unnecessary.

File: cli-tests/tests/output-dir.test.ts [hunk 2]
```diff
@@ -107,22 +123,16 @@ describe("Set of --output-dir tests", () => {
...
 
       // Exit code should be the same
-      xit("solc exit code == zksolc exit code", () => {
-        const solcResult = executeCommand(solcCommand, args);
-        expect(solcResult.exitCode).toBe(result.exitCode);
-
-      });
-      changeDirectoryPermissions(paths.pathToReadOnlyOutputDir, 'a');
-      removeDirectory(paths.pathToReadOnlyOutputDir);
+      // xit("solc exit code == zksolc exit code", () => {
```
Comment (antonbaliasnikov) [inline cli-tests/tests/output-dir.test.ts:130]: Please, remove comments if unnecessary.

**Review by antonbaliasnikov — DISMISSED** [review]

File: cli-tests/tests/output-dir.test.ts (hunk 1 above)
Comment (pcheremu) [reply cli-tests/tests/output-dir.test.ts:115]: This stub should be uncommented after the fix in the zksolc output. The teset itself should contain this code.

File: cli-tests/tests/output-dir.test.ts (hunk 2 above)
Comment (pcheremu) [reply cli-tests/tests/output-dir.test.ts:130]: This stub should be uncommented after the fix in the zksolc output. The teset itself should contain this code.

File: cli-tests/tests/output-dir.test.ts (hunk 1 above)
Comment (antonbaliasnikov) [reply cli-tests/tests/output-dir.test.ts:115]: It's a good practice to add something like:
`// TODO: uncomment after CPR-xxxx is fixed`

File: cli-tests/tests/output-dir.test.ts (hunk 2 above)
Comment (antonbaliasnikov) [reply cli-tests/tests/output-dir.test.ts:130]: https://github.com/matter-labs/era-compiler-solidity/pull/54#discussion_r1532457805

**Review by antonbaliasnikov — APPROVED** [review]

**Comment by hedgar2017** [comment]
@pcheremu should this patch go to zkvyper as well?

**Comment by pcheremu** [comment]
@hedgar2017 Sure, as we agreed before, it will be synced. I will update the test solution for zkvyper based on this patch, of course .

**Pull request merged**